src/dropout.jl

```diff
@@ -155,3 +157,6 @@ _rng_from_array(::AbstractArray) = Random.default_rng()
 
 @non_differentiable _rng_from_array(::Any)
 
+# This exists because `rand!(defalut_rng(), CUDA.rand(3))` ignores the RNG,
+# and Flux would prefer an error. NNlibCUDA will overloard it to produce that.
```

Comment:
```suggestion
# This exists because `rand!(default_rng(), CUDA.rand(3))` ignores the RNG,
# and Flux would prefer an error. NNlibCUDA will overload it to produce that.
```